src/BitPaySDK/Exceptions/RefundUpdateException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace BitPaySDK\Exceptions;
+
+
+class RefundUpdateException extends RefundException
+{
+    private $bitPayMessage = "Failed to update refund";
+    private $bitPayCode    = "BITPAY-REFUND-UPDATE";
+
+    /**
+     * Construct the RefundUpdateException.
+     *
+     * @param string $message [optional] The Exception message to throw.
+     * @param int    $code    [optional] The Exception code to throw.
+     */
+    public function __construct($message = "", $code = 162)
```

Comment:
check other refund exceptions and make sure you add a non existent code numbers, 162 already exists for RefundCreationException